### PR TITLE
Feature/tsr gripper library

### PR DESCRIPTION
**Pull request by athackst**
adding support for generic robots

### CODE REVIEW

**Comment by athackst** [comment]
@mkoval @jeking04 I don't think this will have any adverse affects to current prpy operation, so we could go ahead and merge it in.

File: src/prpy/__init__.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-import base, dependency_manager, logger, ik_ranking, planning, perception, simulation, tsr, viz
+import base, controllers, dependency_manager, logger, ik_ranking, planning, perception, rave, simulation, tsr, viz
```
Comment (mkoval) [inline src/prpy/__init__.py]: Please remove this import - it will cause a regression of #308.

**Comment by mkoval** [comment]
This looks good to me except for the one minor comment above.

@jeking04 Can you also confirm that this won't break anything?

**Comment by jeking04** [comment]
👍  looks good. this is useful additional functionality.

**Comment by athackst** [comment]
updated

**Pull request merged**